source/PlayerInfo.cpp

```diff
@@ -2598,6 +2599,7 @@ void PlayerInfo::UpdateAutoConditions(bool isBoarding)
 			conditions["passenger space"] += ship->Attributes().Get("bunks") - ship->RequiredCrew();
 			++conditions["ships: " + ship->Attributes().Category()];
 		}
+	++conditions["flagship: " + flagship->ModelName()];
```

Comment:
`flagship` could be a nullptr (if the flagship was destroyed in-flight). That is why the code on lines 2604 and 2615 (original numbering) checks if the flagship pointer is valid before de-referencing it.

Reply:
I think 6e036e2 fixes this, though @Amazinite isn't interested in merging this because of #5244 so I'm not sure it matters.

Reply:
My stance has essentially been that once #5244 is in, then we can start going crazy with new auto-conditions if we want (e.g. visited planets/systems, more fleet information like flagship info or fleet size, installed outfits (#5412)).